src/main/java/com/fasterxml/jackson/core/json/async/NonBlockingJsonParser.java

```diff
@@ -194,7 +194,7 @@ public JsonToken nextToken() throws IOException
         _tokenInputTotal = _currInputProcessed + _inputPtr;
         // also: clear any data retained so far
         _binaryValue = null;
-        int ch = _inputBuffer[_inputPtr++] & 0xFF;
+        int ch = getNextByteFromBuffer() & 0xFF;
```

Comment:
If we make this change, might as well add that `& 0xFF` part in accessor.
We can document this in Javadoc (requirement to return "unsigned" byte); plus it's only implemented once per backend.

Reply:
done